src/ansys_sphinx_theme/theme/ansys_sphinx_theme/components/edit-this-page.html

```diff
@@ -0,0 +1,15 @@
+{% if sourcename is defined and theme_use_edit_page_button==true and
+page_source_suffix %} {% set src = sourcename.split('.') %}
+<div class="tocsection editthispage">
+  <a href="{{ fix_edit_link_button(get_edit_provider_and_url()[1]) }}">
+    <i class="fa-solid fa-pencil"></i>
+    {% set provider = get_edit_provider_and_url()[0] %} {% block
+    edit_this_page_text %} {% if provider %} {% trans provider=provider %}Edit
+    on {{ provider }}{% endtrans %} {% else %} {% trans %}Edit{% endtrans %} {%
+    endif %} {% endblock %}
+  </a>
+</div>
+
+{% else %}
+<!-- If there is no MiliSearch enabled, use the PyData search -->
```

Comment:
How is this related to Meilisearch? Are they in conflict?

Reply:
ohh... no i copied it from our meilisearch template, just to get the exact path, i will correct it. sorry for the confusion.